docs/registry.md

```diff
@@ -22,6 +22,22 @@ from a registry, containerd will try these endpoint URLs one by one, and use the
 
 After modify this config, you need restart the `containerd` service.
 
+## Configure Registry TLS Communication
+`cri` plugin also supports configuring TLS settings when communicating with a registry.
+
+To configure the TLS settings for a specific registry, create/modify the `/ec/containerd/config.toml` as follows:
+```toml
+[plugins.cri.registry.tls_configs]
+  [plugins.cri.registry.tls_configs."my.custom.registry"]
```

Comment:
To people who reached here by googling "containerd cri TLS": this `tls_configs` was renamed to `configs` in https://github.com/containerd/cri/commit/53e94c67537b3a33b497817619833c0eb234f6e9#diff-14b27915267f3248ede3980e894f4603 , and the previous form is now simply ignored. (There was no official release with the previous form, so there is no compatibility issue in production)

Reply:
I planned to put that into the release note in the next beta release. Thanks for adding here! :D